nsot/migrations/0027_interface_device_hostname.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('nsot', '0026_model_field_verbose_names'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='interface',
+            name='device_hostname',
+            field=models.CharField(help_text='The hostname of the Device to which the interface is bound. (Internal use only)', max_length=255, db_index=True, blank=True),
```

Comment:
Add `editable=False` here to go along w/ the requested update to the model field.

Run `nsot-server makemigrations --dry-run -v3` afterward just to make sure it says that no changes are detected.